src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -1123,28 +1123,10 @@ internal override void CreateAsync()
                         _processInstance.RunspacePool.Dispose();
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-macos`
Invoke-WebRequest tests.Denial of service.Image Parsing
```powershell
Expected the actual value to be greater than 5, but got 0.38315472351302.
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1932
1932:             $pathologicalRatio | Should -BeGreaterThan 5
```